css/style.css

```diff
@@ -0,0 +1,40 @@
+* {
+  font-size: 24px;
+  font-family: Arial, Helvetica, sans-serif;
```

Comment:
Please, consider setting these properties inside body element

Reply:
When you request for a change on a line(s) of code, kindly give the reason(s), for requesting for such changes.

### :slightly_smiling_face:

Reply:
When you request for a change on a line(s) of code, kindly give the reason(s), for requesting for such changes.

### :slightly_smiling_face: